packages/core/src/internal/clients/cache/AbstractCacheClient.ts

```diff
@@ -190,6 +190,9 @@ export abstract class AbstractCacheClient implements ICacheClient {
     value: string | Uint8Array,
     options?: SetOptions
   ): Promise<CacheSet.Response> {
+    if (typeof options === 'number') {
+      throw new Error('Options must be an object with a ttl property.');
```

Comment:
can you spot check and make sure this is the same error type that we throw if, e.g., a user sets a negative ttl or similar, via those various `validate*` functions?

Reply:
We throw "InvalidArgumentError" on validate* functions. 
Will fix the current pr to use "InvalidArgumentError" error type.